draft-ietf-quic-http.md

```diff
@@ -1384,7 +1384,7 @@ connection error.
 
 Because new error codes can be defined without negotiation (see {{extensions}}),
 receipt of an unknown error code or use of an error code in an unexpected
-context MUST NOT be treated as an error.  However, closing a stream can
+context MUST NOT be treated as a further error.  However, closing a stream can
```

Comment:
Maybe: receipt of an unknown error code or use of an error code in an unexpected context MUST NOT be treated as a connection error.

```suggestion
context MUST NOT be treated as a connection error.  However, closing a stream can
```